cob_hardware_config/robots/cob4-8/urdf/cob4-8.urdf.xacro

```diff
@@ -51,38 +51,33 @@
   <xacro:torso name="torso" parent="base_link" dof1="${False}" dof2="${False}" dof3="${False}">
     <origin xyz="${torso_x} ${torso_y} ${torso_z}" rpy="${torso_roll} ${torso_pitch} ${torso_yaw}" />
   </xacro:torso>
-  <!-- left camera -->
-  <xacro:cob_kinect_v0 name="torso_cam3d_left" ros_topic="torso_cam3d_left" parent="torso_3_link" update_rate="30" data_skip="9">
+  <xacro:realsense_d435 name="torso_cam3d_left" parent="torso_3_link">
     <origin xyz="${torso_cam3d_left_x} ${torso_cam3d_left_y} ${torso_cam3d_left_z}" rpy="${torso_cam3d_left_roll} ${torso_cam3d_left_pitch} ${torso_cam3d_left_yaw}" />
-  </xacro:cob_kinect_v0>
-  <!-- right camera -->
-  <xacro:cob_kinect_v0 name="torso_cam3d_right" ros_topic="torso_cam3d_right" parent="torso_3_link" update_rate="30" data_skip="9">
+  </xacro:realsense_d435>
+  <xacro:realsense_d435 name="torso_cam3d_right" parent="torso_3_link">
     <origin xyz="${torso_cam3d_right_x} ${torso_cam3d_right_y} ${torso_cam3d_right_z}" rpy="${torso_cam3d_right_roll} ${torso_cam3d_right_pitch} ${torso_cam3d_right_yaw}" />
-  </xacro:cob_kinect_v0>
-  <!-- down camera -->
-  <xacro:realsense name="torso_cam3d_down" ros_topic="torso_cam3d_down" parent="torso_3_link">
+  </xacro:realsense_d435>
+  <xacro:realsense_d435 name="torso_cam3d_down" parent="torso_3_link">
     <origin xyz="${torso_cam3d_down_x} ${torso_cam3d_down_y} ${torso_cam3d_down_z}" rpy="${torso_cam3d_down_roll} ${torso_cam3d_down_pitch} ${torso_cam3d_down_yaw}" />
-  </xacro:realsense>
+  </xacro:realsense_d435>
 
-  <xacro:head name="head" parent="torso_3_link" dof1="${True}" dof2="${True}" dof3="${True}">
+  <xacro:head_cad name="head" parent="torso_3_link" dof1="${True}" dof2="${True}" dof3="${True}">
     <origin xyz="${head_x} ${head_y} ${head_z}" rpy="${head_roll} ${head_pitch} ${head_yaw}" />
-  </xacro:head>
+  </xacro:head_cad>
```

Comment:
Please use another name here.
My suggestion: 
- HW Team changes the configuration in the CAD so that `head` is the standard configuration (in CAD as in real life)
- we rename `head_cad` to `head_left` which is special only to Aalto (at the moment)

Reply:
:-1: